index.d.ts

```diff
@@ -178,7 +179,7 @@ ipcMain.on('download-button', async (event, {url}) => {
 ```
 */
 export function download(
-	window: BrowserWindow | BrowserView,
+	window: { webContents: Electron.WebContents },
```

Comment:
This has the downside of not making it clear to the user what they can pass in. Autocomplete is more important than flexibility here I think. So I prefer to just list the supported types.